Packs/SecurityTrails/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### SecurityTrails
```

Comment:
Maybe you should add an explanation about the deletion of 'securitytrails-query-dsl' - just to make it clearer